views/pages/cartoons/about-us.pug

```diff
@@ -0,0 +1,46 @@
+extends /views/templates/frontend-base.pug
+
+block title
+    title #{data.pageTitle} &#8211; MathSoc
+
+append styles
+    link(href="/assets/css/carousel.css" rel="stylesheet")
+    link(href="/assets/css/pages/about-cartoons.css" rel="stylesheet")
+
+append scripts
+    script(src="/assets/js/carousel.js" defer)
+    script(src="https://kit.fontawesome.com/c02fb43ad1.js" crossorigin="anonymous")
+
+block body
+    +banner-image("/assets/img/banners/mathsoc-wall.jpeg")
+    div(class="content")
+        section(class="white")
+            div(class="title")
+                h1 #{data.heading}
+            img(class="cover-pic" src=data.coverPicSrc)
+            div(class="row")
+                div(class="col about-us-section")
+                    h1 #{data.subheading}
+                    p!=data.subheadingCaption
+                    h3 #{data.joinUs}
+                    p(class="app-caption") #{data.applicationsCaption}
+                    div!=data.signupMarkup
+                    div(style="width: 100%; text-align:center;")
+                        +pink-button(data.buttons.team.text, '', data.buttons.team.link)
+                        br
+                        +pink-button(data.buttons.archive.text, '', data.buttons.archive.link)
+                div(class="col")
+                    - var images = data.carouselImages;
+                    +carousel(data.carouselImages)
```

Comment:
looks like the negative margin used to shrink the homepage carousel is now cutting off the top of the image on the cartoons carousel.

![image](https://user-images.githubusercontent.com/37753525/211206770-9f00ba29-2147-4bc5-9fb0-74d3aa02ccae.png)

Reply:
fixed